hphp/tools/tc-print/offline-code.cpp

```diff
@@ -124,7 +121,326 @@ TCA OfflineX86Code::getTransJmpTargets(const TransRec *transRec,
   return aFallThru;
 }
 
-TCA OfflineX86Code::collectJmpTargets(FILE  *file,
+void OfflineCode::printDisasm(TCA startAddr, uint32_t len,
+                                 const vector<TransBCMapping>& bcMap,
+                                 const PerfEventsMap<TCA>& perfEvents,
+                                 bool hostOpcodes) {
+  TCRegion tcr = findTCRegionContaining(startAddr);
+  disasm(tcRegions[tcr].file, tcRegions[tcr].baseAddr, startAddr, len,
+         perfEvents, BCMappingInfo(tcr, bcMap), true, hostOpcodes);
+}
+
+void OfflineCode::loadSymbolsMap() {
+  loadSymbolsMapNm();
+}
+
+void OfflineCode::loadSymbolsMapNm() {
```

Comment:
Hmm, I folded this into the function above a couple months back in 83a65510caa5deea59c59deb69684faaee474d3c. Looks like your rebase lost that, so please fix it.